### PR TITLE
Update the condition in perpendicular_momentum

### PR DESCRIPTION
**Pull request by Jiemin-Li**
As indicated in the title, this is a tiny change about the function of perpendicular_momentum, it should be ready to be merged. @awalter-bnl

### CODE REVIEW

File: src/test_data/arpes.py [hunk 1]
```diff
@@ -363,7 +363,24 @@ def perpendicular_momentum(photon_energy, parallel_momentum,
     $\k_{perpendicular}$ are the surface parallel and perpendicular momentum,
     $E_{ph}$ is the photon energy, $E_{b}$ is the electron binding energy and
     $\Phi$ is the work function.
+    Parameters
+    ----------
+    photon_energy : float or 1D numpy.ndarray,
```
Comment (awalter-bnl) [inline src/test_data/arpes.py]: These first three can also be integers, can you update to include them? Also we should add to all 3 of these 'if an numpy.array then all three need to have the same length or be of length 1.

File: src/test_data/arpes.py [hunk 2]
```diff
@@ -363,7 +363,24 @@ def perpendicular_momentum(photon_energy, parallel_momentum,
...
+        The photon energy (or energies) in eV.
+    parallel_momentum : float or 1D numpy.ndarray,
+        The parallel momentum in the unit of the inverse of angstrom.
+    binding_energy : float or 1D numpy.ndarray,
+        The binding energy (or energies) in eV.
+
+    inner_potential: float or 1D numpy.ndarray
```
Comment (awalter-bnl) [inline src/test_data/arpes.py]: These two values can only be floats or integers (it makes no sense for them to vary by point) can you change this?

File: src/test_data/arpes.py [hunk 3]
```diff
@@ -363,7 +363,24 @@ def perpendicular_momentum(photon_energy, parallel_momentum,
...
+        The inner potential in eV.
+    work_function: float or 1D numpy.ndarray
+        The work function in eV.
 
+    Returns
+    -------
+    $k_z$ : float or 1D numpy.ndarray,
```
Comment (awalter-bnl) [inline src/test_data/arpes.py]: This I think is always a numpy array, never a float. Can you confirm this can be a float or change this?

**Review by awalter-bnl — COMMENTED** [review]
@Jiemin-Li , I have asked for a few changes to the doc-string.

**Comment by Jiemin-Li** [comment]
> @Jiemin-Li , I have asked for a few changes to the doc-string.

Have made changes accordingly. @awalter-bnl

**Comment by awalter-bnl** [comment]
THanks @Jiemin-Li , much appreciated

**Pull request merged**